internal/image/fs.go

```diff
@@ -41,21 +41,17 @@ func historyFromFS(dir *dirs.WindmillDir) (map[refKey][]historyEntry, error) {
 	return result, nil
 }
 
-func historyToFS(dir *dirs.WindmillDir, entriesMap map[refKey][]historyEntry) error {
-	file, err := dir.OpenFile(imagesPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, os.FileMode(0644))
+func addHistoryToFS(dir *dirs.WindmillDir, ref refKey, entry historyEntry) error {
```

Comment:
seems like this would also make sense as a method on a `historyEntry`...?

Reply:
maybe! tho i'd like to keep the serialize/deserialize code in the same place